tests/regForm/regFormPOM.spec.js

```diff
@@ -0,0 +1,123 @@
+
+import { expect, test } from "@playwright/test"
+import HomePage from '../../src/pages/HomePage.js'
+import RegistrationPage from '../../src/pages/RegistrationPage.js'
+import { VALID_USER, INVALID_USER } from '../../src/utils/test-data.js'
+
+test.describe('Registration Form tests', () => {
+    let homePage
+    let registrationPage
+
+    test.beforeEach(async ({ page }) => {
+        homePage = new HomePage(page)
+        registrationPage = new RegistrationPage(page)
```

Comment:
As I said it is not a page but a component

using components directly by tests is not really great idea